docker-compose.yml

```diff
@@ -8,3 +8,37 @@ kafka:
           - zookeeper:zookeeper
         ports:
           - "9092:9092"
+datastore:
+        image: 'mastodonc/kixi.datastore:latest'
```

Comment:
worth checking whether this really updates to latest latest (sometimes you need to force)